SPECS/ding-libs/ding-libs.spec

```diff
@@ -9,38 +9,39 @@
 %global ini_config_version 1.3.1
 Summary:        "Ding is not GLib" assorted utility libraries
 Name:           ding-libs
-Version:        0.6.1
-Release:        45%{?dist}
+Version:        0.6.2
+Release:        55%{?dist}
 License:        LGPLv3+ AND GPLv3+
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://pagure.io/SSSD/ding-libs
```

Comment:
When I went to this URL, it looked like the project fully moved to github; should we update this URL too?

Reply:
I was thinking about that, but the original SPEC, the one I used to understand these changes did keep this URL and I my thought is to minimize the deltas. but I'm open to suggestions.